### PR TITLE
[MO] - [system test] -> user in different namespace with exch messages

### PR DESCRIPTION
**Pull request by Seequick1**
### Type of change

- Bugfix
- Enhancement / new feature
- Refactoring

### Description

System which, verify that UO is able to create users in another namespace and exchange messages. Also fix UserST, missing done().

### Checklist

- [x] Write tests
- [x] Make sure all tests pass

### CODE REVIEW

**Comment by Seequick1** [comment]
@strimzi-ci run tests testcase=AllNamespaceST

File: systemtest/src/test/java/io/strimzi/systemtest/AllNamespaceST.java [hunk 1]
```diff
@@ -86,13 +90,54 @@ void testUOWatchingOtherNamespace() {
...
+
         thirdNamespaceResources.kafkaEphemeral(CLUSTER_NAME, 1)
             .editSpec()
                 .editKafka()
-                    .withNewListeners()
-                        .withNewTls()
-                        .endTls()
+                    .editListeners()
+                        .withNewKafkaListenerExternalNodePort()
+                            .withNewOverrides()
```
Comment (Frawless) [inline systemtest/src/test/java/io/strimzi/systemtest/AllNamespaceST.java]: I think we don't need `override` bootstrap in that case, or do we?

Comment (Seequick1) [reply systemtest/src/test/java/io/strimzi/systemtest/AllNamespaceST.java]: Yes, that's right. We don't need it.

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=regression testcase=AllNamespaceST

**Pull request merged**